var/spack/repos/builtin/packages/foam-extend/package.py

```diff
@@ -72,6 +74,7 @@ class FoamExtend(Package):
     and owner of the OPENFOAM trademark.
     """
 
+    # maintainers = ['??']
```

Comment:
Was this something you want to resolve before this PR gets merged? Or is this a  reminder for future editors? In the latter case IMO it isn't worth keeping here.

Reply:
I'm hoping that someone will step up to base on this. I only put together the openfoam-org package as a courtesy, to ensure the design was consistent with how the openfoam-com and foam-extend packages are done. However, I don't have the time or patience to create installation patches for issues caused by the openfoam-org people. At least with the openfoam-com one I can identify any problems and immediately fix them upstream.

Reply:
Maybe @nrichart is using foam-extend and wants to jump in there (not that he's not otherwise busy). For the openfoam-org one, it may have sounded like @junghans was working with that version.